src/@types/types.d.ts

```diff
@@ -135,6 +136,7 @@ export type ModuleRoute =
       pathname: string;
       exact?: boolean;
       dynamic?: boolean;
+      props?: any;
```

Comment:
Would be better to have Record<string, unknown>, unless we want to allow props: 'foobar' or props: 42.